src/ontology/mondo-edit.obo

```diff
@@ -517605,9 +517607,8 @@ xref: ICD10CM:G04.0 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:
 xref: MEDGEN:1842363 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:592894 {source="MONDO:equivalentTo"}
 xref: UMLS:C5680301 {source="MEDGEN:1842363", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-is_a: MONDO:0019383 {source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"} ! acute disseminated encephalomyelitis
```

Comment:
This line is duplicated with line 517609. It was removed as part of the normalization. It is ok.